query-planner-js/src/config.ts

```diff
@@ -15,4 +17,33 @@ export interface QueryPlannerConfig {
    * Defaults to true.
    */
   reuseQueryFragments?: boolean,
+
+  // Side-note: implemented as an object instead of single boolean because we expect to add more to this soon
+  // enough. In particular, once defer-passthrough to subgraphs is implemented, the idea would be to add a
+  // new `passthroughSubgraphs` option that is the list of subgraph to which we can pass-through some @defer
+  // (and it would be empty by default). Similarly, once we support @stream, grouping the options here will
+  // make sense too.
+  deferStreamSupport?: {
```

Comment:
i'd suggest calling this `incrementalDelivery` or `incrementalDeliverySupport` (that's the more generic name for this feature less focused on the particular set of two directives which are currently envisioned)

Reply:
fixed in #2093, thanks